guides/release-notes.md

```diff
@@ -7,6 +7,19 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New APIs: Hamburger Menu Element
+
+`Wix Editor Elements ($w)`
+
+We added the following new APIs that allow you to customize the [hamburger menu](https://support.wix.com/en/article/editor-x-using-hamburger-menus) element:
```

Comment:
Is editor X the appropriate article to link to here? Is there an alternative for studio?

Reply:
I don't know, but this is the link that Angelique linked to in her documentation and Marta approved it.